app/app.tsx

```diff
@@ -87,6 +92,53 @@ const CustomToast = () => {
   return <Toast config={toastConfig} topOffset={insets.top} />
 }
 
+// Setting up our OTA Updates component
+const OTAUpdates = () => {
+  const [isModalVisible, setIsModalVisible] = useState(false)
+
+  async function fetchAndRestartApp() {
+    await Updates.fetchUpdateAsync()
```

Comment:
might want to still check this result `UpdateFetchResultSuccess` and `UpdateFetchResultFailure`

https://docs.expo.dev/versions/latest/sdk/updates/#updatefetchresult

Then if it fails can alert the user and just not reload.

Something to think about, if it does continuously fail they will have to press "Maybe later" quite a bit. Unless we want to add a don't remind me or something somewhere eventually (or if it failed) to async storage and skip future checks. IDK probably beyond the scope

Reply:
fixed in next push